engine/worker/cmd_cache.go

```diff
@@ -24,11 +24,35 @@ func cmdCache(w *currentWorker) *cobra.Command {
 	cmdCacheRoot := &cobra.Command{
 		Use: "cache",
 		Long: `
-Inside a project, you can create or retrieve a cache from your worker with a tag (useful for vendors for example)
+Inside a project, you can create or retrieve a cache from your worker with a tag (useful for vendors for example).
 
 You can access to this cache from any workflow inside a project. You just have to choose a tag that fits with your needs.
 
 For example if you need a different cache for each workflow so choose a tag scoped with your workflow name and workflow version (example of tag value: {{.cds.workflow}}-{{.cds.version}})
+
+## Use Case
+Java Developers often use maven to manage dependencies. The mvn install command could be long due as there is no existing .m2/ directory on a fresh CDS Job.
+With the worker cache feature, you don't have to wait the download of the dependencies if they didn't been updated on the upstream, since the last run of the job:
```

Comment:
you don't have to download the dependencies if they haven't been updated since the last run of the job